backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -58,6 +58,14 @@ Spree.ready(function(){
   $('.spree_add_fields').click(function() {
     var target = $(this).data("target");
     var new_table_row = $(target + ' tr:visible:last').clone();
+
+    // id of table row
+    new_table_row.attr("id", "spree_new_"+new_table_row.data("hook"));
```

Comment:
`id` of HTML should be unique on the page, can you explain why this is needed to be this way and why we need to set an id for the row?

Reply:
Thats right. But at least we need to remove the id from the persistent copied item. Also we need to mark this row somehow as new nonpersistent item, a (for example in properties spec for check_persisted_property_row_count). We could do that via a class instead. Would that be ok?